src/osd/OSD.cc

```diff
@@ -2512,6 +2513,25 @@ void OSD::create_logger()
 
   PerfCountersBuilder osd_plb(cct, "osd", l_osd_first, l_osd_last);
 
+  // Latency axis configuration for op histograms, values are in nanoseconds
+  PerfHistogramCommon::axis_config_d op_hist_x_axis_config{
+    "Latency (usec)",
+    PerfHistogramCommon::SCALE_LOG2, ///< Latency in logarithmic scale
+    0,                               ///< Start at 0
+    100000,                          ///< Quantization unit is 100usec
```

Comment:
@asheplyakov I think in both HdrHistogram and the implementation I did it's a matter of space usage, if you want better resolution you have to use more buckets. For probabilistic data structures (like Count Min Sketch mentioned before) this could also be adjusted by the amount of allowed noise.